scripts/stack_bounds.py

```diff
@@ -314,24 +359,45 @@ def clean_lines(lines: Iterable[str]) -> Iterator[Tuple[int, str]]:
         elf_txt.close()
 
 
-stack_usage_re = re.compile(r'sp,sp,-(?P<bytes>\d+)')
+ElfInstrUsage = Callable[[ElfInstr], int]
+NodeUsage = Callable[[Node], int]
+
+
+def arm_instr_usage(instr: ElfInstr) -> int:
+    # FIXME
+    return 0
```

Comment:
Would it be better to raise an exception here to fail early if this is invoked for Arm for some reason? Or are we already failing much earlier anyway?

Reply:
Making this an exception would cause the decompile workflow to fail for ARM. As a placeholder, this allowed me to check that the recursion unwinding works for both RISCV64 and ARM, and also allows the decompiler workflow to push jobs through to the backend, which was the immediate need. The placeholder values would cause the back end jobs to fail for ARM, though.

This will only take 20 minutes or so to fix, and I intend to do that before turning on ARM in the backend. I just need to clear a few other todo items first.

Reply:
Ok, no worries. It's also not a big deal to leave the placeholder in here more permanently, was just pointing it out in case it was not intentional.